resin_release_tool/releaser.py

```diff
@@ -21,6 +24,62 @@ def get_canaries(self):
             if tag['tag_key'] == 'CANARY']
         return canaries
 
+    def get_tags_per_device(self):
+        all_devices = self.get_all_devices()
+        tags = self.models.tag.device.get_all_by_application(self.app_id)
+        device_dict = {}
+        for device in all_devices.values():
+            device_id = device['id']
+            device_dict[device_id] = {
+                'uuid': device['uuid'],
+                'tags': {},
+            }
+            self.uuid_list.append(device['uuid'])
+        for elem in tags:
+            device_id = elem['device']['__id']
+            tag_key = elem['tag_key']
+            value = elem['value']
+            device_dict[device_id]['tags'][tag_key] = value
+        tags_per_device = [device_dict[device] for device in list(device_dict.keys())]
+        return tags_per_device
+                    
+    def get_app_env_vars(self):        
+        allvars = self.models.environment_variables.application.get_all(int(self.app_id))
```

Comment:
`allvars` is not snake_case, please modify to `all_vars`.

Reply:
done